scripts/utility.py

```diff
@@ -1280,11 +1272,11 @@ def update_sprite(cat):
         # Don't update the sprite if the cat is faded.
         return
 
-    # First make pelt, if it wasn't possible before
+    """# First make pelt, if it wasn't possible before
```

Comment:
Still needed?

Reply:
I don't believe so, since cats are simply assigned a default "Pelt" rather than None in the define-variable part of __init__. I can remove the commented out code entirely to make things cleaner

Reply:
I would do so. The cleaner the code the better.